tracing/config/tracing.go

```diff
@@ -40,6 +40,28 @@ const (
 	stackdriverProjectIDKey = "stackdriver-project-id"
 )
 
+var (
+	// OnePercentSampling is a configuration that samples 1% of the requests.
+	// TODO(#1712): Remove this and pull "static" configuration from the
```

Comment:
Where does this TODO point to? :thinking:. The configs certainly look like they're not "generic", should they really live here?

Reply:
These are mostly useless now, lemme remove these